core-plugins/src/e2e-test/resources/pluginParameters.properties

```diff
@@ -8,5 +8,19 @@ firstNameCsvFile=testdata/first_name.csv
 lastNameCsvFile=testdata/last_name.csv
 firstNameOutputSchema=[{"key":"id","value":"int"},{"key":"first_name","value":"string"}]
 lastNameOutputSchema=[{"key":"id","value":"int"},{"key":"last_name","value":"string"}]
+errorCollectorOutputSchema=[{"key":"id","value":"int"},{"key":"first_name","value":"string"},\
+  {"key":"msg","value":"string"},{"key":"code","value":"int"},{"key":"node","value":"string"}]
 joinerOutput=testdata/joiner_output.csv
-expectedPartitions=1
+errorCollectorOutput=testdata/error_collector_output.csv
+expectedJoinerOutputPartitions=1
+expectedErrorCollectorOutputPartitions=1
```

Comment:
don't think so
![image](https://user-images.githubusercontent.com/25891353/178550681-45885cb8-5fd6-47e0-ab34-a3b64c511c73.png)

Reply:
For such cases, can we pass expected partitions as `0` indicating we are not sure then it will combine the data from all the output files and sort it and verify?

Reply:
+1